src/arch/ia32/address.rs

```diff
@@ -69,7 +69,7 @@ impl Add for VirtualAddress {
 
 impl AddAssign for VirtualAddress {
     fn add_assign(&mut self, other: Self) {
-        (*self).0 = (*self).0 + other.0;
+        self.0 = self.0 + other.0;
```

Comment:
Is there any reason to not use the operator `+=` ?

Reply:
None. Fixed.